docs/Course Wiki/ECE Course Offerings/ECE330.md

```diff
@@ -0,0 +1,56 @@
+
+This course is graded on an absolute scale, meaning students are graded based on a performance standard, not their performance relative to other students, so doing well on the quizzes and exams is vital to doing well in the class.
+
+## Instructors
+
+Professors Jonathon Schuh, Olga Mironenko, and Richard Zhang have taught the course recently.  The professors all have strong backgrounds in the power field and are very easy to ask questions to.  Going to their office hours is recommended.
+
+## Course Tips
+
+Though it is not required, doing the homework is very important for doing well on the quizzes and exams.  Going to office hours and reading the textbook are helpful for completing the homework and studying for exams.  Many past exams are posted on the course website, and it is recommended for students to go through these exams and their homework problems to study for exams.
+
+## Life After
+
+ECE 330 opens the door to the power and energy field.  You should take this course if you are interested in power generation, electric motors, generators, and drives, transformer design, power electronics, or renewable energy.
+
+[ECE 431 - Electric Machinery](ECE431.md) and [ECE 476 - Power System Analysis](ECE476.md) both have ECE 330 as a prerequisite.  ECE 431 follows on directly from ECE 330 and focuses on transformers and rotating electric machines.  ECE 431 also has a lab component where students make measurements, perform tests, and determine performance characteristics of electric machines.  ECE 476 is a course on power systems, covering topics such as transmission line modeling, power flow, transient stability, and short circuit analysis.  
+
+Students who are interested in power systems should also consider taking [ECE 464 - Power Electronics](ECE464.md) (prerequisite: [ECE 342](ECE342.md)) and [ECE 469 - Power Electronics Laboratory](ECE469.md) (prerequisite: [ECE 343](ECE343.md)), which cover power electronic switching devices and power converters.  ECE 431 is only offered during the spring semester, while ECE 464, ECE 469, and ECE 476 are only offered during the fall semester.
+
+[ECE 333 - Green Electric Energy](ECE333.md) is another great course that is offered every semester. If you are interested in alternative energy such as wind turbines and photo-voltaic cells, you should consider ECE 333. You could also take up Solar Decathlon, which is a two semester course that satisfies your senior design criteria as well. Finally, if you wish to specialize in the control area, you should also take up [ECE 486 - Control Systems](ECE486.md); this course will help you in power electronic design and lead the way to advanced grid system control.
+
+## Infamous Topics
+
+Electromechanical systems, dynamics, state space modeling, linearization, and stability are taught later in the course, and are the most challenging topics.  The textbook is a very helpful resource when it comes to these topics. 
```

Comment:
Make this into a bulleted list.
"- Electromechanical... : these are taught later in the course. the textbook is very helpful..." Also maybe explain why they are so challenging.